app/src/main/java/org/gnucash/android/util/PreferencesHelper.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2015 Alceu Rodrigues Neto <alceurneto@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gnucash.android.util;
+
+import android.preference.PreferenceManager;
+import android.util.Log;
+
+import org.gnucash.android.app.GnuCashApplication;
+
+import java.sql.Timestamp;
+
+/**
+ * A utility class to deal with Android Preferences in a centralized way.
+ */
+public final class PreferencesHelper {
+
+    /**
+     * Should be not instantiated.
+     */
+    private PreferencesHelper() {}
+
+    /**
+     * Tag for logging
+     */
+    protected static final String LOG_TAG = "PreferencesHelper";
+
+
+    private static final String PREFERENCE_LAST_EXPORT_TIME_DEFAULT_VALUE = "PREFERENCE_LAST_EXPORT_TIME_DEFAULT_VALUE";
```

Comment:
Yes. It's used to deal with nothing stored yet. Actually we don't need this
constant. I'll do a little change to make the code more simple and
meaningful.

On Sat, Feb 6, 2016, 12:02 Ngewi Fet <notifications@github.com> wrote:

> In app/src/main/java/org/gnucash/android/util/PreferencesHelper.java
> <https://github.com/codinguser/gnucash-android/pull/475#discussion_r52102348>
> :
>
> > + * A utility class to deal with Android Preferences in a centralized way.
> > + */
> > +public final class PreferencesHelper {
> > +
> > +    /**
> > +     * Should be not instantiated.
> > +     */
> > +    private PreferencesHelper() {}
> > +
> > +    /**
> > +     * Tag for logging
> > +     */
> > +    protected static final String LOG_TAG = "PreferencesHelper";
> > +
> > +
> > +    private static final String PREFERENCE_LAST_EXPORT_TIME_DEFAULT_VALUE = "PREFERENCE_LAST_EXPORT_TIME_DEFAULT_VALUE";
>
> @alceurneto <https://github.com/alceurneto> Should
> PREFERENCE_LAST_EXPORT_TIME_DEFAULT_VALUE not be a time value rather than
> a string constant with the same name as the variable? I think you use it
> later as the default value for a time string right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codinguser/gnucash-android/pull/475/files#r52102348>.
>